README.md

```diff
@@ -122,38 +123,31 @@ Note: the refactoring relations are expressed as indentations
     - [x] Implement Modular inversion.
     - [x] Research about addition chains inversion methods.
   - [x] Add proper tests for every function.
-- [ ] Implement Edwards Points
-    - [ ] Implement Twisted Edwards Extended Coordiantes.
+- [x] Implement Edwards Points
+    - [x] Implement Twisted Edwards Extended Coordiantes.
        - [x] Implement Point Addition.
        - [x] Implement Point Subtraction.
        - [x] Implement Point Doubling.
        - [x] Implement Scalar Mul.
        - [x] Implement from_bytes conversions.
        - [x] Implement to byte conversions.
-       - [ ] Implement compressed Edwards point Y-coordinate.
-    - [ ] Implement Twisted Edwards Projective Coordiates.
+       - [x] Implement compressed Edwards point Y-coordinate.
+    - [x] Implement Twisted Edwards Projective Coordiates.
        - [x] Implement Point Addition.
        - [x] Implement Point Subtraction.
        - [x] Implement Point Doubling.
        - [x] Implement Scalar Mul.
        - [x] Implement from_bytes conversions.
        - [x] Implement to byte conversions.
-       - [ ] Implement compressed Edwards point Y-coordinate.
-    - [ ] Represent Edwards points as Ristretto points using wrapping function (research).
+       - [x] Implement compressed Edwards point Y-coordinate.
+    - [x] Represent Edwards points as Ristretto points using wrapping type or struct.
     - [x] Cargo doc testing and improvement.
     - [x] Decide the best use cases of the various Edwards coordinate types (compressed, standard, extended, projective).
-    - [ ] Benchmark different implementations and algorithms.
-    - [ ] Research About Niels and ProjectiveNiels coordinates usage.
-    - [x] Implement Ristretto Mapping.
-    - [ ] Build and test torsion points.
-    - [ ] Test all point operations for Edwards Points.
```

Comment:
This is quite pedantic but should we make all the 'p's in 'Point' as upper case for the road map? I know it depends on how the implementation is defined but perhaps we could make it all uniform.

Reply:
This can be done before bumping to the next version.